src/libs/actions/Timing.ts

```diff
@@ -42,13 +42,13 @@ function end(eventName: string, secondaryName = '', maxExecutionTime = 0) {
     }
 
     const {startTime, shouldUseFirebase} = timestampData[eventName];
-    Environment.getEnvironment().then((envName) => {
-        const eventTime = Date.now() - startTime;
 
-        if (shouldUseFirebase) {
-            Firebase.stopTrace(eventName);
-        }
+    if (shouldUseFirebase) {
+        Firebase.stopTrace(eventName);
+    }
 
+    const eventTime = performance.now() - startTime;
```

Comment:
NAB: Is it better not to count `Firebase.startTrace(eventName)` and `Firebase.stopTrace(eventName)` in the eventTime? Though both of them look lightweight.

Reply:
ah, good point - for Firebase.startTrace its already the case, I changed it to be also the case for stopTrace!

Reply:
> for Firebase.startTrace its already the case

Hmm, but we should get `startTime` after Firebase.startTrace right?

Reply:
yes, that makes sense as well! I think now we got it 😄

Reply:
ah sorry i think i committed to the wrong branch, one sec

Reply:
okay, now